src/index.tsx

```diff
@@ -7,7 +7,6 @@ import { SketchPicker } from "react-color";
 import { moveOneLeft, moveAllLeft, moveOneRight, moveAllRight } from "./zindex";
 import { roundRect } from "./roundRect";
 import EditableText from "./components/EditableText";
-import { getDateTime } from "./utils";
```

Comment:
We should either dump it here or make a specific file for each function, but having a catch all utils function isn't really helping